lte/gateway/c/session_manager/test/test_local_enforcer.cpp

```diff
@@ -980,22 +980,10 @@ TEST_F(LocalEnforcerTest, test_sync_sessions_on_restart) {
       session_store->get_default_session_update(session_map_2);
   EXPECT_EQ(session_map_2[IMSI1].size(), 1);
 
-  RuleLifetime lifetime1 = {
-      .activation_time = std::time_t(0),
-      .deactivation_time = std::time_t(5),
-  };
-  RuleLifetime lifetime2 = {
-      .activation_time = std::time_t(5),
-      .deactivation_time = std::time_t(10),
-  };
-  RuleLifetime lifetime3 = {
-      .activation_time = std::time_t(10),
-      .deactivation_time = std::time_t(15),
-  };
-  RuleLifetime lifetime4 = {
-      .activation_time = std::time_t(15),
-      .deactivation_time = std::time_t(20),
-  };
+  RuleLifetime lifetime1(std::time_t(0), std::time_t(5));
+  RuleLifetime lifetime2(std::time_t(5), std::time_t(10));
+  RuleLifetime lifetime3(std::time_t(10), std::time_t(15));
+  RuleLifetime lifetime4(std::time_t(15), std::time_t(20));
```

Comment:
hahaha this has been there for long time and i never found the moment to change it
thanks!

Reply:
yeah it's weird I guess GCC can handle it? but clang is stricter, maybe. I was surprised too haha